web_src/js/features/repo-editor.js

```diff
@@ -27,6 +25,7 @@ function initEditPreviewTab($form) {
         mode,
         context,
         text: $form.find(`.tab[data-tab="${$tabMenu.data('write')}"] textarea`).val(),
+        file_path: treePathEl.val(),
```

Comment:
Is this spelling `file_path` correct? I'm not sure if/where this case convertion happens.

Reply:
I didn't try to find where this conversion happens. But I know `filepath` and `filePath` don't work and this one does, because I unsuccessfully tried those first.